app/models/king.rb

```diff
@@ -1,2 +1,20 @@
 class King < ChessPiece
+  X_MOVE_FACTOR = 1
+  Y_MOVE_FACTOR = 1
+
+  def valid_move?(destination_x, destination_y)
+    return false if exceeds_x_movement?(destination_x)
+    return false if exceeds_y_movement?(destination_y)
+    !obstructed?(destination_x, destination_y)
```

Comment:
This is brilliant...the best, most readable version yet IMO. Totally see it now! :+1: